run-election-manager.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
 
+# go to directory where this file is located
+cd "$(dirname "$0")"
```

Comment:
This sometimes gives the wrong results, like when symlinks are involved. I’m guessing it’s fine for us?

Reply:
Yeah I think it's okay, but if you have a better suggestion, I'm not a shell master and welcome it.

Reply:
Here's a SO post on the subject: https://stackoverflow.com/a/246128/549363. The more correct version is… verbose. Let's just go with this for now.